lighty-modules/lighty-netconf-sb/src/test/java/io/lighty/modules/southbound/netconf/tests/NetconfNmdaBaseServiceTest.java

```diff
@@ -188,4 +168,23 @@ public void testBaseServiceEditDataMock() {
         assertNotNull(getSpecificElementSubtree(config, NetconfState.QNAME, "netconf-state"));
     }
 
+
+    private YangInstanceIdentifier getTestSchemaYIID() {
+        YangInstanceIdentifier yangInstanceId = YangInstanceIdentifier.builder()
+                .node(NetconfState.QNAME)
+                .node(Schemas.QNAME)
+                .node(Schema.QNAME)
+                .nodeWithKey(Schema.QNAME, getTestSchemaKeys())
+                .build();
+        return yangInstanceId;
+    }
+
+    private HashMap<QName, Object> getTestSchemaKeys() {
```

Comment:
Remove this method:

1. Make class constants from QNames.
2. Use Map.of(IDENTIFIER, VERSION, FORMAT)

Reply:
Done